libs/core-ui/src/lib/Cohort/ManualCohortManagement/CohortBar.tsx

```diff
@@ -25,6 +25,7 @@ export interface ICohortBarProps {
   cohorts: Cohort[];
   jointDataset: JointDataset;
   modelMetadata: IExplanationModelMetadata;
+  features: unknown[][] | undefined;
```

Comment:
why unknown here? there is no more specific type or set of types we can give here?

Reply:
picking what it is in IDataset, I think it will be either a number or string two D array, if we are sure about it I can change the IDataset
![image](https://user-images.githubusercontent.com/76190371/229152780-099d9835-d81a-4193-a7d5-f83f848a149f.png)